tests/door43_tools_tests/test_templaters.py

```diff
@@ -48,12 +48,18 @@ def testTemplaterTaComplete(self):
         test_file_path = self.extractZipFiles(test_folder_name)
         success = self.doTemplater('ta', test_file_path)
         self.verifyTaTemplater(success, expect_success, self.out_dir,
-                               ['checking.html', 'intro.html', 'process.html', 'translate.html'])
+                               ['01-intro.html', '02-process.html', '03-translate.html', '04-checking.html'])
         # Verify sidebar nav generated
-        soup = BeautifulSoup(read_file(os.path.join(self.out_dir, 'checking.html')), 'html.parser')
+        soup = BeautifulSoup(read_file(os.path.join(self.out_dir, '04-checking.html')), 'html.parser')
         self.assertEqual(len(soup.find('nav', {'id': 'right-sidebar-nav'}).findAll('li')), 49)
         self.assertEqual(len(soup.find('div', {'id': 'content'}).findAll(re.compile(r'h\d+'),
                                                                          {'class': 'section-header'})), 44)
+        # Check that manuals are in order
```

Comment:
@richmahn This new test is also invalid. It does not fail when the code changes in `preprocessors.py` are reverted. 

You should revert the changes to `test_templaters.py` and `en_ta-complete.zip` because they are not related to the code changes you made for this issue.